backend/Services/BathRoomServices.cs

```diff
@@ -0,0 +1,69 @@
+using backend.Services.Interfaces;
+using DTOs;
+
+namespace backend.Services;
+
+public class BathRoomServices : IBathroomService
+{
+    private List<BathroomDTO> _bathroom = new List<BathroomDTO>()
+    {
+        new BathroomDTO()
+        {
+            BathRoomID = Guid.NewGuid(),
+            BathroomQuantity = 2,
+            Shower = true,
+            Toilet = true,
+            DressingTable = true
+        },
+        new BathroomDTO()
+        {
+            BathRoomID = Guid.NewGuid(),
+            BathroomQuantity = 1,
+            Shower = false,
+            DressingTable = true,
+            Toilet = false
+        }
+    };
+
+    public List<BathroomDTO> GetBathRooms()
+    {
+        return _bathroom;
+    }
+
+    public BathroomDTO GetBathRoomById(Guid bathroomID)
+    {
+        return _bathroom.FirstOrDefault(b => b.BathRoomID == bathroomID);
+    }
+
+   
+
+    public bool CreateBathRoom(BathroomDTO bathroomDto)
```

Comment:
create method on services should assure the correct addition of an element

Reply:
Done